src/view/panel/GraphicPool.js

```diff
@@ -127,7 +130,10 @@ Ext.define('BasiGX.view.panel.GraphicPool', {
         // UI displaying the user pictures
         me.pictureView = Ext.create('BasiGX.view.view.GraphicPool', {
             title: false,
-            backendUrls: me.getBackendUrls()
+            backendUrls: me.getBackendUrls(),
+            overItemCls: me.getOverItemCls(),
+            height: me.height,
```

Comment:
```suggestion
            height: me.getHeight(),
```
Ext auto-generates getters for config values, I think it's better to use them instead of directly accessing the value